README.rst

```diff
@@ -30,13 +30,291 @@
 NVIDIA Triton Inference Server
 ==============================
 
-    **NOTE: You are currently on the r20.07 branch which tracks
-    stabilization towards the next release. This branch is not usable
-    during stabilization.**
+    **Triton V2: Starting with the 20.06 release, Triton moves to
```

Comment:
Release 20.07 is the second release of Triton version 2. A legacy V1 version of Triton is available on the r20.07-v1 branch. Triton version 1 is deprecated and no releases beyond 20.07 are planned. More information on the V1 ...

Reply:
Fixed the wording.